src/components/MaximWPproj2/CentCont.styled.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { ButtonStyle, CentreCont, Detailsincont, StyledHeader, StyledNumber } from './styles/Container.styled'
+
+type Props = {
+  title: string
+  number: string
+  text1: string
+  text2: string
+  text3: string
+}
+
+export const CentCont = (props: Props) => {
+  return (
+    <CentreCont>
+      <StyledHeader containertype={'Main'}>{props.title}</StyledHeader>
+      <StyledNumber containertype={'Main'}>{props.number}</StyledNumber>
+      <Detailsincont containertype={'Main'}>
+        <hr style={{ width: 250 }} />
```

Comment:
should be styled in the styled container. this file is not suppose to have .styled in the file name